pkgs/applications/audio/alsa-scarlett-gui/default.nix

```diff
@@ -24,6 +24,10 @@ stdenv.mkDerivation rec {
   sourceRoot = "source/src";
   nativeBuildInputs = [ pkg-config wrapGAppsHook4 ];
   buildInputs = [ gtk4 alsa-lib ];
+  postInstall = ''
+      substituteInPlace $out/share/applications/vu.b4.${pname}.desktop \
+        --replace "Exec=/bin/alsa-scarlett-gui" "Exec=$out/bin/${pname}"
```

Comment:
Intendation is not matching 2 spaces which is written in our contributing guide and also we shouldn't substitute pname wherever possible

Reply:
https://github.com/NixOS/nixpkgs/pull/246636